gr-blocks/include/gnuradio/blocks/correctiq_auto.h

```diff
@@ -0,0 +1,51 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_CORRECTIQ_CORRECTIQ_AUTO_H
+#define INCLUDED_CORRECTIQ_CORRECTIQ_AUTO_H
+
+#include <gnuradio/blocks/api.h>
+#include <gnuradio/sync_block.h>
+
+namespace gr {
+namespace blocks {
+
+/*!
+ * \brief <+description of block+>
```

Comment:
a short word on how this block differs from the general one, and from the other implementations, would be most helpful here.